wiki/SimRel/Simultaneous_Release_Requirements.md

```diff
@@ -289,15 +220,24 @@ see [EPL License Feature](https://github.com/eclipse-cbi/epl-license-feature/blo
 
 ### Re-use and share common third party code
 
-Any third-party plug-ins that are common between projects must be consumed via [Eclipse Orbit](https://github.com/eclipse-orbit/.github/blob/main/profile/README.md).
-The Simultaneous Release must not have duplicate third-party libraries.
+Any third-party plug-ins that are common between projects must be consumed via 
+[Eclipse Orbit](https://github.com/eclipse-orbit/.github/blob/main/profile/README.md),
+or if available from Maven Central as an OSGi artifact,
+maybe be consumed directly from Maven Central,
+in which case it should be PGP signed and **not** Jar signed.
+Third party libraries must not be repackaged as bundles
+and distributed with a bundle symbolic name
+that has the potential to conflict with a bundle symbolic name used by another  project.
+
+The Simultaneous Release must avoid unnecessary duplicate third-party libraries.
```

Comment:
👍 

@merks: Could we add a link here to the SimRel content overview that you prepare for M1/M2 where participating projects can easily see the current state of things?